### PR TITLE
Adds toy dual arm robot and changes naming in test.

### PR DESCRIPTION
**Pull request by maxspahn**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "urdfenvs"
-version = "0.1.8"
```
Comment (GijsGroote) [inline pyproject.toml:3]: Nice, I see automatic version control works!

File: urdfenvs/dual_arm/envs/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+__pycache__/
```
Comment (GijsGroote) [inline urdfenvs/dual_arm/envs/.gitignore:1]: I feel like tests/.pytest_cache should also be added. What do you think?

**Review by GijsGroote — APPROVED** [review]
Mostly tops, my only tip is:
The changes could be 2 commits, One for the dual_arm and one for the changes in tests.

File: pyproject.toml (hunk 1 above)
Comment (maxspahn) [reply pyproject.toml:3]: I did it manually :(

File: urdfenvs/dual_arm/envs/.gitignore (hunk 2 above)
Comment (maxspahn) [reply urdfenvs/dual_arm/envs/.gitignore:1]: Why?

**Pull request merged**